tests/api/test_workflow_tag.py

```diff
@@ -0,0 +1,127 @@
+from collections import namedtuple
+
+import pytest
+from starlette.testclient import TestClient
+from fastapi import status
+
+from agr_literature_service.api.main import app
+from agr_literature_service.api.models import WorkflowTagModel, ReferenceModel
+from .fixtures import auth_headers, db # noqa
+from .test_reference import test_reference # noqa
+from .test_mod import test_mod # noqa
+
+TestWTData = namedtuple('TestWTData', ['response', 'new_wt_id', 'related_ref_curie', 'related_mod_id',
+                                       'related_mod_abbreviation'])
+
+
+@pytest.fixture
+def test_workflow_tag(db, auth_headers, test_reference, test_mod): # noqa
+    print("***** Adding a test workflow tag *****")
+    with TestClient(app) as client:
+        new_wt = {"reference_curie": test_reference.json(),
+                  "mod_abbreviation": test_mod.new_mod_abbreviation,
+                  "workflow_tag_id": "ont1",
+                  }
+        response = client.post(url="/workflow_tag/", json=new_wt, headers=auth_headers)
+        yield TestWTData(response, response.json(), test_reference.json(), test_mod.new_mod_id,
+                         test_mod.new_mod_abbreviation)
+
+
+class TestWorkflowTag:
+
+    def test_get_bad_ref_ont(self):
```

Comment:
ref_ont is kind of confusing, why not workflow_tag ?  It works though, so it's fine.  Same at 66-70 and the new method definitions.

Reply:
the old unit tests used _ont but I agree that it is confusing. I'll change them to _wt